.golangci.yml

```diff
@@ -69,9 +104,9 @@ issues:
     - path: _test\.go
       linters:
         - errcheck
-        - gofmt
+        # - gofmt
```

Comment:
why disabled?

Reply:
Why comment them out instead of removing them?

Reply:
i'm ok with deleting comments, just wanted to make sure that we didn't accidentally forget to ignore that excluded path on any linters. If we keep comments we can see that we enabled them there on purpose vs just forgetting to exclude them. Doesn't matter much.